code/modules/clothing/suits/hood.dm

```diff
@@ -95,3 +95,35 @@
 			suit.RemoveHood()
 		else
 			qdel(src)
+
+/obj/item/clothing/head/hooded/screened_niqab/red
```

Comment:
As the entire idea behind the clothes is to completely conceal you, I suggest adding the following lines of code here so it automatically blocks any hair
```suggestion
/obj/item/clothing/head/hooded/screened_niqab
    flags = BLOCKHAIR

/obj/item/clothing/head/hooded/screened_niqab/red
```